tcpdf.php

```diff
@@ -15266,11 +15266,11 @@ public function getBarcode() {
 	 * Print a Linear Barcode.
 	 * @param string $code code to print
 	 * @param string $type type of barcode (see tcpdf_barcodes_1d.php for supported formats).
-	 * @param int $x x position in user units (empty string = current x position)
-	 * @param int $y y position in user units (empty string = current y position)
-	 * @param int $w width in user units (empty string = remaining page width)
-	 * @param int $h height in user units (empty string = remaining page height)
-	 * @param float $xres width of the smallest bar in user units (empty string = default value = 0.4mm)
+	 * @param int|null $x x position in user units (null = current x position)
```

Comment:
`float|null`

`$x`, `$w` accepts `float` in general

Reply:
Not sure it's worth reviewing since I got no answer from @nicolaasuni so far :s

Reply:
It is, he only do the merges a few times per year :D :(

Reply:
Then feel free to list me the int|null to change to float|null, I'll do the update ;)

Reply:
basically any location/dimension parameter

Reply:
```suggestion
	 * @param float|null $x x position in user units (null = current x position)
```

Reply:
> basically any location/dimension parameter

I just listed them all then. Can you confirm before I commit them ?

Reply:
LGTM 👍